common/source/docs/common-msp-osd-overview-4.2.rst

```diff
@@ -41,16 +41,138 @@ Features
 Configuration
 -------------
 
-To enable MSP OSD, set the following parameters ( example using SERIAL port 2 as the port to attach to the DJI Air unit using both TX and RX lines):
+To enable telemetry based MSP OSD, set the following parameters ( example using SERIAL port 2 as the port to attach to the DJI Air unit using both TX and RX lines):
 
  - :ref:`OSD_TYPE<OSD_TYPE>` = 3 if no integrated OSD is being used in order to activate the OSD code. If an integrated OSD is present and the user wishes to have both OSDs , then :ref:`OSD_TYPE<OSD_TYPE>` = 1 will activate the on-board OSD as well as providing screens for the MSP OSD function. For example, on vehicles using the DJI goggles/air system for medium range, but still running a long range VTX using the internal OSD for when the vehicle exceeds the range of the HD DJI Goggles. This configuration could use one OSD screen optimized for DJI Goggles, and another for the integrated OSD and the user can switch between them depending on which video system is being viewed.
  - :ref:`SERIAL2_PROTOCOL<SERIAL2_PROTOCOL>` = 33
- - :ref:`MSP_OPTIONS<MSP_OPTIONS>` = 0 (polling mode)
+ - :ref:`SERIAL2_BAUD<SERIAL2_BAUD>` = 115
+ - :ref:`MSP_OPTIONS<MSP_OPTIONS>` bit 0 = 1 (EnableTelemetryMode)
```

Comment:
this is incorrect....this is push mode for MSP OSDs using only the TX line not dji goggles

Reply:
Ok - I have to say I don't really understand the MSP_OPTIONS bit 1 then, all I know is that if I set it to 1 my HDZero goggles don't work. So when would you ever set it to 1?

Reply:
on an msp osd that is connected only by its rx input to the uarts tx output...normally the osd requests that info be semt..this option allows it to be 'pushed' to the osd

Reply:
So the bit itself "EnableTelemetryMode" - should actually be "EnableTelemetryPollingMode" ?

Reply:
> this is incorrect....this is push mode for MSP OSDs using only the TX line not dji goggles

So which goggles would this apply to?

Reply:
a standalone osd like mwosd

Reply:
So if I say this: 
.. note:: a standalone osd like MWOSD, with only one wire connected, will need :ref:`MSP_OPTIONS<MSP_OPTIONS>` bit 0 = 1 (EnableTelemetryMode). This is the only case where this should be used.

Reply:
I am working on a rewrite to make the various osd types clearer..standby